test/utils/vulnerability.test.ts

```diff
@@ -39,7 +39,7 @@ describe('Vulnerability utils', () => {
       const cmdExceptions = [1165, 1890];
       expect(consoleStub.called).to.equal(false);
       const result = getExceptionsIds(NSPRC, cmdExceptions);
-      expect(result).to.have.length(8).and.deep.equal([1165, 1890, 985, 1213, 1654, 2000, 2001, 2100]);
+      expect(result).to.have.length(7).and.deep.equal([1165, 1890, 985, 1213, 2000, 2001, 2100]);
```

Comment:
mock data had some data points that had gone past the set expiration date, requiring the test fixes in this file